setup/README.adoc

```diff
@@ -36,7 +36,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OpenShiftSDN
+  networkType: OVNKubernetes
```

Comment:
just curious - why this change ? I'm still using `OpenShiftSDN` for my "local" development. Did something changed in ocp 4.15 ?

Reply:
Ha got it! Thanks for the heads up. I still need to switch to 4.15 locally so that's why I haven't faced this issue.

Reply:
Also last times I've used ClusterBot to generate Rosa cluster, which turned out to be quicker to provision and more "stable" in kind of installation process.

Reply:
:thinking: but our clusters are still running OpenShiftSDN, right? Or did they started migrating the already existing clusters to OVNKubernetes? IIRC, the migration was no-go for the already existing OSD clusters some time ago, so I'm curious if this has changed as well or not.

Reply:
Yeah, my understanding is that migration to OVNKubernetes is still not supported for OSD clusters. At least according to the 4.15 docs: https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html

We should check with the OSD team though. I mentioned it to @alexeykazakov already.